# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1294863447 18000
#      Wed Jan 12 15:17:27 2011 -0500
# Node ID a339b7a7f73bbdfaf3b771910851b74359545060
# Parent  d86389a6dc3f19ab000569807885c02b493a2ee2
rand.cc: delete unused static function

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-12  John W. Eaton  <jwe@octave.org>
+
+	* DLD-FUNCTIONS/rand.cc (reset_rand_generator): Delete unused
+	static function.
+
 2011-01-12  John W. Eaton  <jwe@octave.org>
 
 	* graphics.cc (graphics_object::get_ancestor): Avoid GCC warning.
 
 2011-01-12  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/gcd.cc (extended_gcd): Tag call to floor with gnulib::.
 
diff --git a/src/DLD-FUNCTIONS/rand.cc b/src/DLD-FUNCTIONS/rand.cc
--- a/src/DLD-FUNCTIONS/rand.cc
+++ b/src/DLD-FUNCTIONS/rand.cc
@@ -482,22 +482,16 @@ using the \"reset\" keyword.\n\
 %!   assert(skewness(x),0,0.012); 
 %!   assert(kurtosis(x),-6/5,0.0094);
 %! endif
 */
 
 
 static std::string current_distribution = octave_rand::distribution ();
 
-static void
-reset_rand_generator (void)
-{
-  octave_rand::distribution (current_distribution);
-}
-
 DEFUN_DLD (randn, args, ,
   "-*- texinfo -*-\n\
 @deftypefn  {Loadable Function} {} randn (@var{x})\n\
 @deftypefnx {Loadable Function} {} randn (@var{n}, @var{m})\n\
 @deftypefnx {Loadable Function} {} randn (\"state\", @var{x})\n\
 @deftypefnx {Loadable Function} {} randn (\"state\", \"reset\")\n\
 @deftypefnx {Loadable Function} {} randn (\"seed\", @var{x})\n\
 @deftypefnx {Loadable Function} {} randn (\"seed\", \"reset\")\n\
